mscore/file.cpp

```diff
@@ -311,6 +311,7 @@ Score* MuseScore::openScore(const QString& fn)
 
       MasterScore* score = readScore(fn);
       if (score) {
+            score->updateCapo();
```

Comment:
I suppose this might be not the best place to call this. `openScore` is not called, for example, when opening a score from a command line or a file manager. Wouldn't `readScore` or some other function be a better place for this?